feature-libs/asm/core/facade/asm.service.spec.ts

```diff
@@ -20,25 +22,47 @@ const mockUser: User = {
   customerId: '123456',
 };
 
+class AsmConnectorMock {
+  customerSearch(
+    _options: CustomerSearchOptions
+  ): Observable<CustomerSearchPage> {
+    return of();
+  }
+  bindCart(_cartId: string, _customerId: string): Observable<unknown> {
```

Comment:
please add unit test for the new function `bindCart`.

Reply:
See `feature-libs/asm/occ/adapters/occ-asm.adapter.ts`.